salt/thorium/check.py

```diff
@@ -14,6 +14,21 @@ def gt(name, value):
     '''
     Only succeed if the value in the given register location is greater than
     the given value
+
+    USAGE::
```

Comment:
Can you switch all of these `USAGE::`s to just `USAGE:`? The `::` makes the `.. code-block:: yaml` formatting not add the correct `yaml` tag on the doc examples and formats the docs inconsistently.